### ISSUE TITLE
Firmware 2.4.27

### ISSUE THREAD

**Issue by Jeremiah164** [peacey/udm-telus#12]
This does not work on firmware 2.4.27. Unifi changed the Dreammachine Pro to be more like the SE

**Comment by tylerdola** [comment]
Still works for me and I just updated to 3.0.19. The path locations are different though, there is no /mnt, just install direct to /data/igmpproxy  and /data/on_boot.d

**Comment by tylerdola** [comment]
![image](https://user-images.githubusercontent.com/14016834/229320823-0a79c86a-f43d-47ba-a194-ebdedacfd048.png)
Actually I just updated to 3.0.19 and Network 7.4.149 and there is a built in IGMP Proxy via the IPTV support. No more need for this wonderful udm-telus.  @peacey, appreciate all that you have done.

**Comment by tylerdola** [comment]
To clarify/add: I still have the IPTV vlan network set up, but the firewall rules were defined for me when I check off the IPTV support, I can disable/delete the firewall rules defined in the instructions found here via peacey
![image](https://user-images.githubusercontent.com/14016834/229322697-a736ebd6-ae31-4fee-8997-f4793f9b04b2.png)

![image](https://user-images.githubusercontent.com/14016834/229322701-19188de7-139a-4fa5-a5b3-171ceef88fd0.png)

**Comment by tylerdola** [comment]
I had it working with this for a while and the igmpproxy shut off. It stopped working after a few hours, so I'm going to blame Ubiquiti and go back to the original set up. 


*EDIT:* Its working with stock Unifi OS 3.0.19 with the built in IGMP proxy. I just had to make sure IGMP snooping was turned off.

**Comment by dslawdog** [comment]
> I had it working with this for a while and the igmpproxy shut off. It stopped working after a few hours, so I'm going to blame Ubiquiti and go back to the original set up.
> 
> _EDIT:_ Its working with stock Unifi OS 3.0.19 with the built in IGMP proxy. I just had to make sure IGMP snooping was turned off.

Just wanted to reach out and see if you've had continued success using the IPTV support button (vs all the config peacey did).

**Comment by tylerdola** [comment]
> > I had it working with this for a while and the igmpproxy shut off. It stopped working after a few hours, so I'm going to blame Ubiquiti and go back to the original set up.
> > _EDIT:_ Its working with stock Unifi OS 3.0.19 with the built in IGMP proxy. I just had to make sure IGMP snooping was turned off.
> 
> Just wanted to reach out and see if you've had continued success using the IPTV support button (vs all the config peacey did).

Yes, it is still working! But I still followed peacey's awesome instructions for the VLAN creation. I just checked off the new IPTV support checkbox and ensured that IPTV vlan did not have IGMP snooping enabled.

**Comment by christohill** [comment]
@tylerdola Was there anything specific needed to get the IPTV Support checkbox to appear? I'm not seeing it and can't seem to figure out how to enable it.

Current versions UDM-SE (both say up to date):
Unifi OS: 3.0.18
Network: 7.3.83

![image](https://github.com/peacey/udm-telus/assets/2646218/effb7eb2-74d3-4a0f-b199-ee4c9658059c)

**Comment by Jeremiah164** [comment]
> @tylerdola Was there anything specific needed to get the IPTV Support checkbox to appear? I'm not seeing it and can't seem to figure out how to enable it.
> 
> Current versions UDM-SE (both say up to date): Unifi OS: 3.0.18 Network: 7.3.83
> 
> ![image](https://user-images.githubusercontent.com/2646218/237536241-effb7eb2-74d3-4a0f-b199-ee4c9658059c.png)

You need to be in early access to get it. I was able to get IPTV working great using this https://github.com/fabianishere/udm-iptv
